code/CMSBatchActionHandler.php

```diff
@@ -280,11 +280,11 @@ protected function buildAction($class)
      */
     protected function cleanIDs($csvIDs)
     {
-        $ids = preg_split('/ *, */', trim($csvIDs));
+        $ids = preg_split('/ *, */', trim($csvIDs ?? ''));
         foreach ($ids as $k => $id) {
             $ids[$k] = (int)$id;
         }
-        return array_filter($ids);
+        return array_filter($ids ?? []);
```

Comment:
Correct fix here would be some defensive programming like adding a new line about this such as `$ids = is_array($ids) ? $ids : [];`

However since this hasn't come up in any unit test failures, I think we should just leave it as is.  (There will be thousands of other examples similar to this)